docs/Glossary/index.md

```diff
@@ -0,0 +1,143 @@
+
+The collective group of all [Peers](#peer) of a given torrent (this includes both [Seeders](#seeder) and [Leechers](#leecher)) who are actively sharing any portion of the torrent.
+
+### Torrent File
+
+A file containing metadata and integrity information about the files and folders associated with the torrent, along with a list of the network locations for its [trackers](#tracker). Torrent files typically bear the extension `.torrent`.
+
+### Tracker
+
+Servers that assist [Peers](#peer) in locating each other and forming efficient distribution groups known as [swarms](#swarm). The term 'tracker' is also informally used for sites that host [.torrent files](#torrent-file) and operate their own tracker servers.
+
+### Perma-Seed
+
+Seeding your torrents permanently ("forever"). Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.
+
+---
+
+## Usenet
+
+### Atomic Moves
+
+This is a real file move, carried out instantaneously, rather than by copying a file from one location to another and then deleting the file from the original folder.
+
+### Instant Moves
+
+Same as [Atomic Moves](#atomic-moves).
+
+### PAR (PAR2) Files
+
+PAR and PAR2 files are "Parity Archive Volume" files primarily used in Usenet as a means to attempt repairing incomplete or corrupted files. The presence of these files, however, is not always a guarantee that "reconstruction" of the original data is possible.
+
+### DMCA
+
+Also known as "DMCA takedown". The DMCA Takedown process allows creators to send notices to service providers that have material that infringes their copyright.
+When a DMCA takedown request is issued to a [USP (Usenet Service Provider)](#usp), you may end up with missing or incomplete parts, and extraction can subsequently fail.
+
+[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act){:target="\_blank" rel="noopener noreferrer"}
+
+### N&T
+
+Also known as "Notice&Takedown". The European equivalent of [DMCA](#dmca).
+
+[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Notice_and_take_down){:target="\_blank" rel="noopener noreferrer"}
+
+### USP
+
+Short for **U**senet **S**ervice **P**rovider
+
+Usenet providers maintain the servers that store Usenet's content and offer you access to download files. [Indexers](#indexers) are required to search for files on Usenet and can help you locate specific files within the network.
+
+### Indexers
+
+An indexer is a search engine from which you can download NZB files. NZB files point to the locations of all the different parts that make up an entire file or release. Most indexers require registration and some are invite-only.
+
+When used in the context of Radarr/Sonarr, indexers can refer to both Usenet indexers and [torrent trackers](#tracker).
+
+---
+
+## Shorthand
```

Comment:
I suggest removing this section and adding it within the definitions

Reply:
I agree, I typically don't change anchors when I do grammar, but I think if we were going to have this be its own section at all, something more like "Acroynms and Alternate Identifiers" or something. I dunno, shorthand seems a little weird to use in this context, but I have no real strong opinion on it. Just aesthetics.

Reply:
Happy to remove this and make them part of each definition. Will do that in a subsequent pass.